src/main/java/roomescape/domain/user/Member.java

```diff
@@ -0,0 +1,81 @@
+package roomescape.domain.user;
+
+import jakarta.persistence.*;
+
+import java.util.Objects;
+
+@Entity
+public class Member {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Embedded
+    private Name name;
+    @Embedded
+    private Email email;
+    @Embedded
+    private Password password;
+    @Enumerated(EnumType.STRING)
+    private Role role;
```

Comment:
JPA 엔티티를 작성하다보면 지금처럼 어노테이션이 다양하게 붙는 상황이 주로 생기곤 합니다(DTO도 마찬가지고요). 이런 경우 개행만 적절히 해준다면 코드의 가독성을 좀 더 높일 수 있어요 :)

```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    
    @Embedded
    private Name name;
    
    @Embedded
    private Email email;
    
    @Embedded
    private Password password;
    
    @Enumerated(EnumType.STRING)
    private Role role;
```

Reply:
멤버변수는 항상 당연히 붙여왔는데 이렇게 어노테이션이 많은 경우엔 개행을 추가하는 편이 훨씬 가독성이 좋네요! 🙇